bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/InstGmosCommon.java

```diff
@@ -1280,6 +1281,17 @@ public static boolean isNodAndShuffleableObsType(Config c) {
     private static final CategorizedTime DHS_WRITE = CategorizedTime.fromSeconds(Category.DHS_WRITE, 10.0);
 
     public CategorizedTimeGroup calc(Config cur, Option<Config> prev) {
+        final ItemKey FPU_KEY = new ItemKey("instrument:fpu");
+        final double FPU_OVERHEAD = 60.0;
+
+        final ItemKey FILTER_KEY = new ItemKey("instrument:filter");
+        final double FILTER_OVERHEAD = 20.0;
+
+        final ItemKey DISPERSER_KEY = new ItemKey("instrument:disperser");
+        final double DISPERSER_OVERHEAD = 90.0;
```

Comment:
I guess these constants should go somewhere else but not sure where.

Reply:
Let's define them with their respective definitions like we do in `Flamingos2`.  Because of the whacky way that GMOS enums are split over north and south versions, I think this means in the interface definitions like `GmosCommonType.FPUnit` etc.

Reply:
I was reluctant to touch those GMOS enums because I needed to reshuffle several types around and was worried about breaking something fundamental in the core. But I'll give it a try, I'll be counting on your review for reassurance 😄 .